src/Authentication.Abstractions/AzureEnvironment.cs

```diff
@@ -459,6 +467,11 @@ public AzureEnvironment(IAzureEnvironment other)
         /// </summary>
         public string ContainerRegistryEndpointSuffix { get; set; }
 
+        /// <summary>
+        /// The token audience required for communicating with the Log Analytics service in this enviornment
+        /// </summary>
+        public string OperationalInsightsEndpointResourceId { get; set; }
+        
```

Comment:
OperationalInsightsEndpointResourceId is defined in extendedProperties. I believe this defination is redundant.

Reply:
Yes, I agree. Removing redundancy.